protocol/contracts/farm/facets/MarketplaceFacet/Dynamic.sol

```diff
@@ -136,6 +141,10 @@
         return PPoly32(ranges, values, bases, f.signs, f.mode);
     }
 
+    /**
+    * @notice Creates a 'zero set' of PPoly32 values.
+    * @dev To be used when hashing constant listings/orders to prevent hashing collisions
+    */
     function createZeros() internal pure returns (uint256[32] memory ranges, uint256[128] memory values, uint256[4] memory bases) {
```

Comment:
I don't actually believe we need this to prevent hashing collisions. Its something we can bring up to Halborn to be 100% confident but as long as what we are encoding/hashing doesn't have arguments that have variable LENGTHS  we are OK. Having a variable NUMBER of arguments is alright. Can cover this tmrw